### ISSUE TITLE
Need SRA run accession on Variation service

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#2019]
We provide it on assembly, so I think we should provide it on the services where our users would like to see it (RNA-Seq, Variation, Metagenomic Binning).

**Comment by dawenx** [comment]
@aswarren do you think the back-end can support SRA? if not, are you planning to support it?

**Comment by dawenx** [comment]
@ARWattam if back-end does not support SRA, i suggest to close this issue and add **wontfix** label. 

ps. i will make another pass on outstanding issue like this that assigned to me in a week or so and **close** them if there's no update...

**Comment by ARWattam** [comment]
But it should do it!  We have enabled it for RNA-Seq, Assembly, Metagenomic Read Mapping, Taxonomic Classification.  I don’t understand why we can’t do this for Variation.

**Comment by dawenx** [comment]
@ARWattam i agree. sorry i mentioned wrong person (to give him a push), let me try again: @aswarren

**Comment by aswarren** [comment]
There is an SRA-import utility that needs to be invoked in the Variation pipeline script.
Relevant links are
https://github.com/PATRIC3/sra_import
https://github.com/PATRIC3/app_service/blob/master/scripts/App-Variation.pl
https://github.com/PATRIC3/app_service/blob/master/app_specs/Variation.json

With examples at
https://github.com/aswarren/Prok-tuxedo/blob/master/prok_tuxedo.py#L319-L322
https://github.com/PATRIC3/app_service/blob/master/app_specs/RNASeq.json#L107-L127

**Comment by dawenx** [comment]
@olsonanl Bob, can you help on the back-end side? Thanks!

**Comment by dawenx** [comment]
@AllanDickerman hi Allan, can you help on the back-end side? an early bird told me that you are the chose one. :)
